upgrade/1.2/Stage_0_Prerequisites.md

```diff
@@ -270,6 +270,11 @@ If the following command does not complete successfully, check if the `TOKEN` en
    [Upgrade Troubleshooting](README.md#relevant-troubleshooting-links-for-upgrade-related-issues).
    If the failure persists, then open a support ticket for guidance before proceeding.
 
+   `prerequisites.sh` clears the existing CFS configuration for each Management node. As each
+   Management node is upgraded, documentation will refer to the CFS configuration that should be
```

Comment:
I don't think this is accurate. It is true that CFS is cleared for all Management nodes, but our docs do not say anything about CFS as the nodes are upgraded. We handle CFS as the final step of the upgrade process.

I see that this PR is adding this in stage 2 for the worker nodes, but not for the master nodes (also in stage 2) or storage nodes (in stage 1). Should we also be adding callouts for CFS for those nodes as they are upgraded, or should we revise this paragraph to be clear that we only call out the CFS stuff as the worker nodes are upgraded.

Reply:
The "documentation will refer to the CFS configuration" part was a reference to the other blurb I am adding with this PR, specifically the callout in Stage 2 to the GSG which tells them how to put the config back in place.  And then, like you said, existing Stage 5 (IIRC) documentation goes into detail on the CSM configuration.

I am unaware of any critical CFS configuration that needs to be applied to master and storage nodes in this "gap" that exists between the time when CSM upgrades wipe configuration and later in the overall upgrade process when each product adds their configuration layer back in.  The point of this doc update is to cover this gap period so the HSN and DVS on worker nodes continue to function.  If someone is aware of CFS layers that are critical for master and/or storage nodes, I am happy to expand the documentation.  Internal testing, AFAIK, was done with the worker change only.

Reply:
So in that case I'd change the wording of this paragraph. Something like:
```suggestion
   worker node is upgraded, the documentation will refer to the CFS configuration that should be
```